code/datums/station_traits/positive_traits.dm

```diff
@@ -270,3 +270,60 @@
 		item.add_fingerprint(living_mob, ignoregloves = TRUE)
+
+/datum/station_trait/cybernetic_revolution
+	name = "Cybernetic Revolution"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	weight = 1
+	report_message = "The new trends in cybernetics have come to the station! Everyone has some form of cybernetic implant."
+	trait_to_give = STATION_TRAIT_CYBERNETIC_REVOLUTION
+	/// List of all job types with the cybernetics they should receive.
+	var/static/list/job_to_cybernetic = list(
+		/datum/job/assistant = /obj/item/organ/internal/heart/cybernetic, //real cardiac
+		/datum/job/atmospheric_technician = /obj/item/organ/internal/cyberimp/mouth/breathing_tube,
+		/datum/job/bartender = /obj/item/organ/internal/liver/cybernetic/tier3,
+		/datum/job/botanist = /obj/item/organ/internal/cyberimp/chest/nutriment,
+		/datum/job/captain = /obj/item/organ/internal/heart/cybernetic/tier3,
+		/datum/job/cargo_technician = /obj/item/organ/internal/stomach/cybernetic/tier2,
+		/datum/job/chaplain = /obj/item/organ/internal/cyberimp/brain/anti_drop,
+		/datum/job/chemist = /obj/item/organ/internal/liver/cybernetic/tier2,
+		/datum/job/chief_engineer = /obj/item/organ/internal/cyberimp/chest/thrusters,
+		/datum/job/chief_medical_officer = /obj/item/organ/internal/cyberimp/chest/reviver,
+		/datum/job/clown = /obj/item/organ/internal/cyberimp/brain/anti_stun, //HONK!
+		/datum/job/cook = /obj/item/organ/internal/cyberimp/chest/nutriment/plus,
+		/datum/job/curator = /obj/item/organ/internal/eyes/robotic/glow,
+		/datum/job/detective = /obj/item/organ/internal/lungs/cybernetic/tier3,
+		/datum/job/doctor = /obj/item/organ/internal/cyberimp/arm/surgery,
+		/datum/job/geneticist = /obj/item/organ/internal/fly, //we don't care about implants, we have cancer.
+		/datum/job/head_of_personnel = /obj/item/organ/internal/eyes/robotic,
+		/datum/job/head_of_security = /obj/item/organ/internal/eyes/robotic/thermals,
+		/datum/job/janitor = /obj/item/organ/internal/eyes/robotic/xray,
+		/datum/job/lawyer = /obj/item/organ/internal/heart/cybernetic/tier2,
+		/datum/job/mime = /obj/item/organ/internal/tongue/robot, //...
+		/datum/job/paramedic = /obj/item/organ/internal/cyberimp/eyes/hud/medical,
+		/datum/job/prisoner = /obj/item/organ/internal/eyes/robotic/shield,
+		/datum/job/psychologist = /obj/item/organ/internal/ears/cybernetic/upgraded,
+		/datum/job/quartermaster = /obj/item/organ/internal/stomach/cybernetic/tier3,
+		/datum/job/research_director = /obj/item/organ/internal/cyberimp/bci,
+		/datum/job/roboticist = /obj/item/organ/internal/cyberimp/eyes/hud/diagnostic,
+		/datum/job/scientist = /obj/item/organ/internal/ears/cybernetic,
+		/datum/job/security_officer = /obj/item/organ/internal/cyberimp/arm/flash,
+		/datum/job/shaft_miner = /obj/item/organ/internal/monster_core/rush_gland,
+		/datum/job/station_engineer = /obj/item/organ/internal/cyberimp/arm/toolset,
+		/datum/job/virologist = /obj/item/organ/internal/lungs/cybernetic/tier2,
```

Comment:
Similarly, does this create air-breathing plasmamen?

Reply:
probably